libraries/AP_OSD/AP_OSD.cpp

```diff
@@ -278,3 +281,21 @@ void AP_OSD::next_screen()
     } while (t != current_screen && !screen[t].enabled);
     current_screen = t;
 }
+
+void AP_OSD::stats()
+{
+   float alt;
+    AP::ahrs().get_relative_position_D_home(alt);
+    alt = -alt;
+    AP_AHRS &ahrs = AP::ahrs();
+    Vector2f v = ahrs.groundspeed_vector();
+   float speed;
+   speed = v.length();
+   bool flying;
+   flying = (AP_Notify::flags.armed) && ((speed > 2) || (alt > 5));
```

Comment:
Check will fail with copter holding the position but still flying.

Reply:
added throttle check